### DISCUSSION TITLE
Game Crashes while playing halfway through the RTS minigame in Fort Condor.

### DISCUSSION THREAD

**Discussion by TheDreamweaver1189AD** [julianxhokaxhiu/FFNx#546] [Q&A] [answered]
It was working yesterday with no problem, I played today, and then the game Crashed when playing the minigame halfway through:

"Feel free to visit this link to know about further next steps you can take
https://github.com/julianxhokaxhiu/FFNx/blob/master/docs/faq.md"

**Comment by julianxhokaxhiu** [comment, marked as answer]
I'm sorry but reports like this are not really useful. Additionally you might probably be playing using 7th, so I'd suggest you using their support channels for any help. Cheers and thanks for using FFNx :)
